### ISSUE TITLE
Auto login does not trigger in certain conditions

### ISSUE THREAD

**Issue by gdestuynder** [mozilla-iam/auth0-custom-lock#122]
1. make sure you cookies for phonebook.mozilla.org are deleted
2. make sure you have auto login working (login to sso.mozilla.com for example)
3. Access https://phonebook.mozilla.org/#search/kang

Results:
NLX asks for manual login

Expected results:
Be automatically logged in

Note: going to phonebook.mozilla.org will auto-login properly

This was reported on behalf of @jvehent

**Comment by jvehent** [comment]
haie. I am security resaircheur from france. do I get bounty money for zis bogue?

**Comment by gdestuynder** [comment]
In the NLX URL you can find this: `&state=qAVnbWveG6r6f_1QuIcwk5d-m5EvMSws#search/kang&` (note this state is no longer valid) which shows there's a parsing issue somewhere

**Comment by gene1wood** [comment]
This also occurs for this RP
```
https://mozdef.private.scl3.mozilla.com:9443/app/kibana#/discover
```

**Comment by gdestuynder** [comment]
Note in the case of this url i didnt see any garbled field but it didnt have the correct params in the /authorize call.
it could be that nlx on dev looked different when i ran my first tests
in both cases though auto login fails

**Comment by gdestuynder** [comment]
Fixed by #126 and deployed
